context/AuthContext.js

```diff
@@ -1,19 +1,49 @@
-import React, { children, createContext, useState } from "react";
+import React, { createContext, useState, useEffect } from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
```

Comment:
earlier the app was not able to store the userToken so every time the app reloaded the user was prompted to go through the onboarding part again, used asyncstorage to sore the userToken value